File: platforms/kotlin/examples/scene/model/visible.kt
import org.fragmentcolor.*

val mesh = Mesh()
mesh.addVertex(Vertex.pbr(listOf(0.0f, 0.5f, 0.0f)))
val model = Model(mesh, Material.pbr()?)

// Models start visible; toggle with """set_visible""".
val visible_now = model.visible()

File: platforms/python/examples/scene/model/visible.py
from fragmentcolor import Material, Mesh, Model, Vertex

mesh = Mesh()
mesh.add_vertex(Vertex.pbr([0.0, 0.5, 0.0]))
model = Model(mesh, Material.pbr())

# Models start visible; toggle with `set_visible`.
visible_now = model.visible()